src/services/helpdesk/hooks/index.js

```diff
@@ -1,5 +1,7 @@
 const { authenticate } = require('@feathersjs/authentication');
 const globalHooks = require('../../../hooks');
+const userModel = require('../../user/model');
```

Comment:
Codacy found an issue: ['userModel' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5822205)